packages/api/src/payments/payrexxPaymentProvider.ts

```diff
@@ -125,7 +125,15 @@ export class PayrexxPaymentProvider extends BasePaymentProvider {
         method: 'GET'
       }
     )
-    if (res.status !== 200) throw new Error(`Payrexx response is NOK with status ${res.status}`)
+    if (res.status !== 200) {
+      logger('payrexxPaymentProvider').error(
+        res,
```

Comment:
Is logging whole `res` is actually enough ? I believe body will be `readableStream` and it's not gonna log it's content but rather object. Please confirm you've checked it.
Otherwise you can write `status`, `statusText` and possibly `body()`.